### PR TITLE
Remove trailing newline character if exists

### PR DESCRIPTION
**Pull request by Farzat07**

### CODE REVIEW

File: edit.py [hunk 1]
```diff
@@ -60,6 +60,9 @@ def read_file(path, buf):
         with open(PATH) as f:
             text = f.read()
 
+        if text[-1] == "\n":
+            text = text[:-1]  # remove trailing newline if exists.
+
```
Comment (keith) [inline edit.py]:
```suggestion
            text = f.read().strip()
```

hrm should we just `.strip()` instead? probably fine to not allow any leading space either?

Comment (Farzat07) [reply edit.py]: I don't know... If leading space is included, wouldn't that be intentional?
Same if multiple trailing newlines are included. Strip would remove them all, while [:-1] only removes one, allowing trailing newlines to be included intentionally if desired.
Both work though I guess...

Comment (keith) [reply edit.py]: i think it should be ok 🙃

**Comment by keith** [comment]
Thanks!

**Pull request merged**